content/fr/docs/setup/pick-right-solution.md

```diff
@@ -0,0 +1,302 @@
+application facile à installer pour votre environnement Mac ou Windows qui vous permet de
+commencer à coder et déployer votre code dans des conteneurs en quelques minutes sur un nœud unique Kubernetes.
+
+* [Minishift](https://docs.okd.io/latest/minishift/) installe la version communautaire de la plate-forme d'entreprise OpenShift 
+de Kubernetes pour le développement local et les tests. Il offre une VM tout-en-un (`minishift start`) pour Windows, macOS et Linux,
+ le `oc cluster up` containerisé (Linux uniquement) et [est livré avec quelques Add Ons faciles à installer] (https://github.com/minishift/minishift-addons/tree/master/add-ons).
+
+* [MicroK8s](https://microk8s.io/) fournit une commande unique d'installation de la dernière version de Kubernetes sur une machine locale
+pour le développement et les tests. L'installation est rapide (~30 sec) et supporte de nombreux plugins dont Istio avec une seule commande.
+
+* [IBM Cloud Private-CE (Community Edition)](https://github.com/IBM/deploy-ibm-cloud-private) peut utiliser VirtualBox sur votre machine
+pour déployer Kubernetes sur une ou plusieurs machines virtuelles afin de développer et réaliser des scénarios de test. Cette solution
+peut créer un cluster multi-nœuds complet.
+
+* [IBM Cloud Private-CE (Community Edition) sur Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers) est un script IaC (Infrastructure as Code) basé sur Terraform/Packer/BASH pour créer un cluster LXD à sept nœuds (1 Boot, 1 Master, 1 Management, 1 Proxy et 3 Workers) sur Linux Host.
+
+* [Kubeadm-dind](https://github.com/kubernetes-sigs/kubeadm-dind-cluster) est un cluster Kubernetes multi-nœuds (tandis que minikube est
+un nœud unique) qui ne nécessite qu'un docker-engine. Il utilise la technique du docker-in-docker pour engendrer le cluster Kubernetes.
+
+* [Ubuntu sur LXD](/docs/getting-start-guides/ubuntu/local/) supporte un déploiement de 9 instances sur localhost.
+
+## Solutions hebergées
+
+* [AppsCode.com](https://appscode.com/products/cloud-deployment/) fournit des clusters Kubernetes managés pour divers clouds publics, dont AWS et Google Cloud Platform.
+
+* [APPUiO](https://appuio.ch) propose une plate-forme de cloud public OpenShift, supportant n'importe quel workload Kubernetes. De plus, APPUiO propose des Clusters OpenShift privés et managés, fonctionnant sur n'importe quel cloud public ou privé.
+
+* [Amazon Elastic Container Service for Kubernetes](https://aws.amazon.com/eks/) offre un service managé de Kubernetes.
+
+* [Azure Kubernetes Service](https://azure.microsoft.com/services/container-service/) offre des clusters Kubernetes managés.
+
+* [Containership Kubernetes Engine (CKE)](https://containership.io/containership-platform) Approvisionnement et gestion intuitive de clusters
+ Kubernetes sur GCP, Azure, AWS, Packet, et DigitalOcean. Mises à niveau transparentes, mise à l'échelle automatique, métriques, création de 
```

Comment:
`mise à l'échelle automatique` looks weird.
I think we can simply use `auto-scaling`, public cloud providers like IBM and AWS are using `auto-scaling` in their FR dashboards.

Reply:
100% agree, auto-translation glitch, thanks